piton/tools/pli/iop/bw_lib.cc

```diff
@@ -0,0 +1,212 @@
+// Modified by Princeton University on June 9th, 2015
```

Comment:
Do we want to just make this a symbolic link for now instead of having a separate file? I can't decide about it. Either that or we could just `git mv` the file but we need to make sure all the makefiles are updated if we do that

Reply:
I think having separate files the way it is right now, while slightly redundant, would be best in case some files require the .c version as dependencies or some future code needs the .c instead of .cc.

Reply:
Need for this was removed thanks to a fix to global.h